requirements.txt

```diff
@@ -1,4 +1,4 @@
-pyfrc>=2018.2.3
+pyfrc>2019.0.0
```

Comment:
your less than sign and my less than sign look different

Reply:
But <2019.0.0 isn't entirely the correct answer. Think about why not

Reply:
Do you mean because it would include previous years? Does the `requirements.txt` format support giving a range of versions?

Reply:
it does indeed. You can do pyfrc>=2018.0.0,<2019.0.0